src/components/header.js

```diff
@@ -85,9 +85,9 @@ const Header = ({ siteTitle }) => (
       <Link to="/gifts" activeStyle={activeStyle}>
         Gifts
       </Link>
-      <Link to="/misc" activeStyle={activeStyle}>
+      {/* <Link to="/misc" activeStyle={activeStyle}>
```

Comment:
@Rakhee06  Its not good practice to keep commented out code without an explanation. So either add a comment or delete the code going forward. Thank you! :)